test/recipes/30-test_evp_data/evppkey_slh_dsa_keygen.txt

```diff
@@ -16,6 +16,14 @@ Ctrl = hexseed:AC379F047FAAB2004F3AE32350AC9A3D829FFF0AA59E956A87F3971C4D58E7100
 CtrlOut = hexpub:0566D240CC519834322EAFBCC73C79F5A4B84F02E8BF0CBD54017B2D3C494B57
 CtrlOut = hexpriv:AC379F047FAAB2004F3AE32350AC9A3D829FFF0AA59E956A87F3971C4D58E7100566D240CC519834322EAFBCC73C79F5A4B84F02E8BF0CBD54017B2D3C494B57
 
+FIPSversion = >=3.6.0
+KeyGen = SLH-DSA-SHA2-128s
+KeyName = tcId1-security-category
+Ctrl = hexseed:AC379F047FAAB2004F3AE32350AC9A3D829FFF0AA59E956A87F3971C4D58E7100566D240CC519834322EAFBCC73C79F5
+CtrlOut = hexpub:0566D240CC519834322EAFBCC73C79F5A4B84F02E8BF0CBD54017B2D3C494B57
+CtrlOut = hexpriv:AC379F047FAAB2004F3AE32350AC9A3D829FFF0AA59E956A87F3971C4D58E7100566D240CC519834322EAFBCC73C79F5A4B84F02E8BF0CBD54017B2D3C494B57
+Security-Category = 1
```

Comment:
Do you remember that these are slow to run ??..

Reply:
Yes I do remember.  The alternative is to not test this case.  I figured slow was the lesser evil.
These cannot be done using an existing test case without breaking the FIPS cross version checks.

Happy to drop all of these if you want a different tradeoff.